src/biogeochem/CNBalanceCheckMod.F90

```diff
@@ -214,7 +221,7 @@ subroutine CBalanceCheck(this, bounds, num_soilc, filter_soilc, &
          write(iulog,*)'wood_harvestc            = ',wood_harvestc(c)*dt
          write(iulog,*)'grainc_to_cropprodc      = ',grainc_to_cropprodc(c)*dt
          write(iulog,*)'-1*som_c_leached         = ',som_c_leached(c)*dt
-         call endrun(msg=errMsg(sourcefile, __LINE__))
+         call endrun(msg=errMsg(sourcefile, __LINE__))   !zgdu
```

Comment:
Please remove `!zgdu` comment. Please correct in other places.

Reply:
Done.